packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -153,6 +154,15 @@ const App = (props) => {
         location?.pathname
     )
 
+    // TODO: Remove test logs before merging
+    if (isServer) {
+        logger.log('log test message')
+        logger.info('info test message', {key:'retail-react-app', details:['App']})
```

Comment:
The tickets asks us to provide documentation to the users. What would this doc look like?

Reply:
Basic best practices for logging, like using the correct log level when using console.log so we can filter the appropriate log level in MRT.

We can also document this logger class if users want to use it in their project.